metriq/models/method.py

```diff
@@ -67,4 +46,4 @@ class Methods(Page):
         results (List[Method]): List of methods on this page.
     """
 
-    results: List[Method]
+    results: List[Method]
```

Comment:
New line at end-of-file, please. (That's the only "extraneous" white space change we ever want.)

Reply:
Done.